packages/edit-widgets/src/index.js

```diff
@@ -27,3 +28,19 @@ export function initialize( id, settings ) {
 		document.getElementById( id )
 	);
 }
+
+/**
+ * Initializes the block editor in the widgets Customizer section.
+ *
+ * @param {string} id       ID of the root element to render the section in.
+ * @param {Object} settings Block editor settings.
+ */
+export function customizerInitialize( id, settings ) {
```

Comment:
>This would assume that widgets behavior is pretty common across the customizer and widgets screens.

It is.

>One potential downside to this is that we load this entire module for both the widgets and customizer screens, even if large parts of code which is loaded isn't actually relevant for the current screen.

Only 2 small files are not relevant, everything else is the same. Do you think there is something that could be pruned there?